machine/accton/accton_as7716_32x/kernel/mgmt-port-init-config.patch

```diff
@@ -0,0 +1,26 @@
+Initail configurations for management port
+
+diff --git a/drivers/net/ethernet/broadcom/tg3/tg3.c b/drivers/net/ethernet/broadcom/tg3/tg3.c
+index 87a7911..4ba76b8 100644
+--- a/drivers/net/ethernet/broadcom/tg3/tg3.c
++++ b/drivers/net/ethernet/broadcom/tg3/tg3.c
+@@ -1628,6 +1628,7 @@ static void tg3_mdio_config_5785(struct tg3 *tp)
+ static void tg3_mdio_start(struct tg3 *tp)
+ {
+ 	tp->mi_mode &= ~MAC_MI_MODE_AUTO_POLL;
++	tp->mi_mode |= MAC_MI_MODE_SHORT_PREAMBLE;
+ 	tw32_f(MAC_MI_MODE, tp->mi_mode);
+ 	udelay(80);
+ 
+@@ -2899,6 +2900,11 @@ static int tg3_phy_reset(struct tg3 *tp)
+ 		}
+ 	}
+ 
++	if (tp->phy_id == TG3_PHY_ID_BCM5718S) {
++		__tg3_writephy(tp, 0x8, 0x10, 0x1d0);
++		__tg3_writephy(tp, 0x1f, 0x4, 0x5e1);
++	}
```

Comment:
The management port MAC which has internal PHY (0x8) connects to external PHY (0x1f).  ``mgmt-port-init-config.patch`` has the following initializations:

1. set internal PHY to SGMII mode that can connect to external PHY
2. set MAC_MI_MODE_SHORT_PREAMBLE for accessing external PHY
3. set basic auto-negotiation capabilities for external PHY